lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -326,11 +340,28 @@ sub engine_workit {
         my $error = locate_local_assets(\%vars, $assetkeys);
         return $error if $error;
     }
+    my $casedir             = OpenQA::Utils::testcasedir($vars{DISTRI}, $vars{VERSION}, $shared_cache);
+    my $productdir          = OpenQA::Utils::productdir($vars{DISTRI}, $vars{VERSION}, $shared_cache);
+    my $relative_casedir    = substr($casedir,          length($vars{PRJDIR}) + 1);
+    my $relative_productdir = substr($productdir,       length($vars{PRJDIR}) + 1);
+    my $target_name         = substr($relative_casedir, 0, index($relative_casedir, '/'));
 
     $vars{ASSETDIR}   //= OpenQA::Utils::assetdir();
-    $vars{CASEDIR}    //= OpenQA::Utils::testcasedir($vars{DISTRI}, $vars{VERSION}, $shared_cache);
-    $vars{PRODUCTDIR} //= OpenQA::Utils::productdir($vars{DISTRI}, $vars{VERSION}, $shared_cache);
-
+    $vars{CASEDIR}    //= $relative_casedir;
+    $vars{PRODUCTDIR} //= $relative_productdir;
+
+    # if the CASEDIR is a git repository, we do not need to do symlink for CASEDIR.
+    # The code will be git clone in os-autoinst.
+    if ($vars{CASEDIR} !~ /http(s)?\:\/\//) {
+        my $link_casedir_error = _link_testrepo(catdir($vars{PRJDIR}, $target_name) , $pooldir, $target_name);
+        return $link_casedir_error if $link_casedir_error;
+    }
```

Comment:
Did you introduce this just to not create an unused symlink? I would prefer if openQA would as of now not care at all about the "git clone feature" within os-autoinst. That os-autoinst might not use the symlink for needles should not be our concern here

Reply:
yes, my purpose was that. I removed that `if` block.